examples/deploy_service/main.tf

```diff
@@ -21,6 +21,7 @@ locals {
 provider "google" {
   credentials = "${file(var.credentials_path)}"
   region      = "${var.region}"
+  version     = "~> 1.20"
```

Comment:
These version-pins seem like a separate issue to me. Thoughts on extracting to a separate PR?

Reply:
Agreed, extracted to https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/90